src/templates/blog.jsx

```diff
@@ -67,3 +68,7 @@ export const query = graphql`
 `;
 
 export default BlogTemplate;
+
+export const Head = () => (
+  <SEO title={SEO_DATA.blog.title} description={SEO_DATA.blog.description} />
```

Comment:
Just for consistency, I suggest spreading SEO_DATA as well. 
```suggestion
  <SEO {...SEO_DATA.blog} />
```